### ISSUE TITLE
Align all selection margins

### ISSUE THREAD

**Issue by Skaiir** [bpmn-io/form-js#424]
__Describe the Bug__

To avoid more conflicts in the CSS than we are already going to have, I've decided to take on the alignment of selection margins as part of a separate ticket here.

The issue in question is best described visually:

![image](https://user-images.githubusercontent.com/17801113/203383848-f41ecf40-28a2-4457-91ed-6a15d3c84893.png)
![image](https://user-images.githubusercontent.com/17801113/203383921-59237d51-65ed-4438-9057-5edafa0e3b8b.png)
![image](https://user-images.githubusercontent.com/17801113/203383941-75bf824f-74fd-420c-a03b-e704ebf570de.png)
![image](https://user-images.githubusercontent.com/17801113/203384015-e9d23a64-776f-441b-83b1-bfe2e4f9f1d0.png)

There is inconsistency in the paddings applied to various fields. We would like to homogenize this a little bit. 

__Steps to Reproduce__

1. Select various components of a form in the editor
2. Notice the variation in padding, especially bottom padding

__Expected Behavior__

All padding top and bottom should be equal in all scenarios. 

__Related__

https://github.com/bpmn-io/form-js/issues/406

**Comment by adrianmarinwork** [comment]
Do you want me to take care of this?

**Comment by Skaiir** [comment]
@adrianmarinwork Actually in retrospect this is waiting on a few components to be merged so the good first issue label may have not been appropriate, thank you for offering help. Another one that could use external contribution would be [this issue here](https://github.com/bpmn-io/form-js/issues/412), should be something quite simple to get started on this codebase.
